order.php

```diff
@@ -33,67 +25,52 @@
                             </qrcode>
                           </a>
                 </div>
-                <div class="bnomics-qr-code-hint"><?=__('Click on the QR code to open in the wallet', 'edd-blockonomics')?></div>
+                <div class="bnomics-qr-code-hint">
+                  <a href="bitcoin:{{order.address}}?amount={{order.satoshi/1.0e8}}" target="_blank"><?=__('Open in wallet', 'edd-blockonomics')?></a>
+                </div>
               </div>
               <!-- BTC Amount -->
               <div class="bnomics-amount">
               <div class="bnomics-bg">
                 <!-- Order Status -->
                 <div class="bnomics-order-status-wrapper">
-                  <span class="bnomics-order-status-title" ng-show="order.status == -1" ng-cloak ><?=__('To confirm your order, please send the exact amount of <strong>BTC</strong> to the given address', 'edd-blockonomics')?></span>
+                  <span class="bnomics-order-status-title" ng-show="order.status == -1" ng-cloak ><?=__('To pay, send exactly this BTC {{altcoinselect}} amount')?></span>
                   <span class="warning bnomics-status-warning" ng-show="order.status == -3" ng-cloak><?=__('Payment Expired (Use the browser back button and try again)', 'edd-blockonomics')?></span>
                   <span class="warning bnomics-status-warning" ng-show="order.status == -2" ng-cloak><?=__('Payment Error', 'edd-blockonomics')?></span>
                 </div>
-                    <h4 class="bnomics-amount-title" for="invoice-amount" ng-hide="order.status == -3">
-                     {{order.satoshi/1.0e8}} BTC
-                    </h4>
-                    <div class="bnomics-amount-wrapper" ng-hide="order.status == -3">
-                      <hr class="bnomics-amount-seperator"> ≈
-                      <span ng-cloak>{{order.value}}</span>
-                      <small ng-cloak>{{order.currency}}</small>
+                    <div>
```

Comment:
The checkout fields do not hide when the payment expires
![EDD Payment expired](https://user-images.githubusercontent.com/39666372/105029112-93504580-5a5a-11eb-9670-c9983a30b2c8.png)

Reply:
Fixed